src-schema/PhpunitTestCase.php

```diff
@@ -73,6 +73,8 @@ public function stopQuery(): void
 
     protected function tearDown(): void
     {
+        $this->db->connection->connection()->close();
```

Comment:
make sure this is also never merged in

Reply:
This explicit diconnection solved the issue.

There is no circular reference, simply up to now the number of tests was below the Postres limit.

Reply:
I already explain on Discord, that this is no true, otherwise burn testing will fail.

Reply:
try replacing this with `gc_collect_cycles()` by yourself :)